docker/addons/provision/docker-compose.yml

```diff
@@ -25,11 +25,9 @@ services:
       MF_PROVISION_LOG_LEVEL: ${MF_PROVISION_LOG_LEVEL}
       MF_PROVISION_HTTP_PORT: ${MF_PROVISION_HTTP_PORT}
       MF_PROVISION_CONFIG_FILE: ${MF_PROVISION_CONFIG_FILE}
-      MF_PROVISION_LOG_LEVEL: ${MF_PROVISION_LOG_LEVEL}
```

Comment:
Why are these removed? Not used?

Reply:
@drasko It is declared twice (three line above)